tests/dags/providers/provider_api_scripts/test_metropolitan_museum.py

```diff
@@ -6,181 +6,164 @@
-    expect_json = {"it": "works"}
-    endpoint = "https://abc.com"
-    query_params = {"a": "b"}
-    retries = 2
-    with patch.object(
-        mma.delayed_requester, "get_response_json", return_value=expect_json
-    ) as mock_get_response_json:
-        r_json = mma._get_response_json(query_params, endpoint, retries=retries)
-
-    assert r_json == expect_json
-    mock_get_response_json.assert_called_once_with(
-        endpoint, query_params=query_params, retries=retries
-    )
+# abbreviated response without other images 45733
+single_object_response = json.loads(
+    (RESOURCES / "sample_response_without_additional.json").read_text()
+)
+# single expected record if 45733 with no additional images
+single_expected_data = json.loads((RESOURCES / "sample_image_data.json").read_text())
+
+# response for objectid 45734 with 2 additional image urls
+full_object_response = json.loads((RESOURCES / "sample_response.json").read_text())
+# 3 expected image records for objectid 45734
+full_expected_data = json.loads(
+    (RESOURCES / "sample_additional_image_data.json").read_text()
+)
 
 
-def test_create_meta_data():
-    exact_response = {
-        "accessionNumber": "36.100.45",
-        "classification": "Paintings",
-        "creditLine": "The Howard Mansfield Collection, Purchase, Rogers Fund, 1936",
-        "culture": "Japan",
-        "objectDate": "late 17th century",
-        "medium": "Hanging scroll; ink and color on silk",
-    }
-    exact_meta_data = {
-        "accession_number": "36.100.45",
-        "classification": "Paintings",
-        "credit_line": "The Howard Mansfield Collection, Purchase, Rogers Fund, 1936",
-        "culture": "Japan",
-        "date": "late 17th century",
-        "medium": "Hanging scroll; ink and color on silk",
-    }
-    r = requests.Response()
-    r.status_code = 200
-    r.json = MagicMock(return_value=exact_response)
-    with patch.object(mma.delayed_requester, "get", return_value=r):
-        response = mma._get_response_json(None, "", retries=2)
-        meta_data = mma._create_meta_data(response)
+CC0 = LicenseInfo(
+    "cc0", "1.0", "https://creativecommons.org/publicdomain/zero/1.0/", None
+)
+
 
-    assert exact_meta_data == meta_data
+@pytest.mark.parametrize(
+    "test_date, expected",
+    [("2022-07-01", {"metadataDate": "2022-07-01"}), (None, None)],
+)
+def test_get_next_query_params(test_date, expected):
+    ingester = MetMuseumDataIngester(test_date)
+    actual = ingester.get_next_query_params()
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "response_json, expected",
+    [
+        ({"total": 4, "objectIDs": [153, 1578, 465, 546]}, [153, 1578, 465, 546]),
+        (None, None),
+    ],
+)
+def test_get_batch_data(response_json, expected):
+    actual = mma.get_batch_data(response_json)
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "case_name, response_json, expected",
+    [
+        (
+            "single image",
```

Comment:
Do you think it would be reasonable to add the empty case for this and the next 3 tests? E.g. `response_json` is `{}` or `None`